src/lsp/cobol_ast/terms.ml

```diff
@@ -615,13 +630,20 @@ module FMT = struct
     | BXor -> "B-XOR"
   and pp_binop ppf o = string ppf (show_binop o)
 
+  and pp_binary_relation ppf (a, o, b) =
+    fmt "%a@ %s@ %a" ppf
+      pp_expression a ([%derive.show: relop] o) pp_expression b
```

Comment:
What is `[%derive.show: relop]` ? Why not use the `pp_relop` generated by `deriving show` ? it's not the same ?

Reply:
Indeed, it's less opaque.